testsuite/features/step_definitions/command_steps.rb

```diff
@@ -593,6 +593,11 @@
   output, _code = $server.run('/usr/sbin/rhn-search cleanindex', check_errors: false)
   log "Search reindex finished." if output.include?("Index files have been deleted and database has been cleaned up, ready to reindex")
   raise 'The output includes an error log' if output.include?('ERROR')
+  repeat_until_timeout(timeout: 5000, message: "waiting for search index to become active again") do
+    result, code = $server.run('systemctl is-active rhn-search')
+    break if result.include? "active"
```

Comment:
```suggestion
    break if result.include? 'active'
```

Ruby code style suggests to use double quotes only if you have a variable inside the string, otherwise, just simple quotes.